pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
To clean this up, can we just perform these escalation checks in the authorizer itself? That would avoid having a separate registry